spec/support/build_test_app.rb

```diff
@@ -63,7 +63,7 @@ def ensure_scaffold
         scaffold_dir.rmtree unless @keep_scaffold
         self
       rescue => e
-        binding.pry
+        # binding.pry
```

Comment:
If you're removing the breakpoint (which makes sense!) you probably want to remove the blanket rescue as well?